src/transformers/models/mobilebert/modeling_tf_mobilebert.py

```diff
@@ -88,6 +88,37 @@
 ]
 
 
+class TFMobileBertPreTrainingLoss:
+    """
+    Loss function suitable for BERT-like pretraining, that is, the task of pretraining a language model by combining
+    NSP + MLM. .. note:: Any label of -100 will be ignored (along with the corresponding logits) in the loss
+    computation.
+    """
+
+    # Copied from transformers.models.bert.modeling_tf_bert.TFBertPreTrainingLoss.hf_compute_loss
```

Comment:
Why not put the copied from above? How is the rest different?

Reply:
Fixed!